src/common/post.ts

```diff
@@ -158,7 +164,7 @@ export type CreatePost = Pick<
   'title' | 'content' | 'image' | 'contentHtml' | 'authorId' | 'sourceId' | 'id'
 >;
 
-export const saveFreeformPost = async (
+export const createFreeformPost = async (
```

Comment:
Change to better show for what this function is used for since if it was also for updating it would need separate logic for updating flags (to avoid overwrite).